### DISCUSSION TITLE
Inverter support

### DISCUSSION THREAD

**Discussion by ghost** [Sleeper85/esphome-jk-bms-can#4] [General]
Hi Sleeper85. 
I want to thank you for your good job. Thank you for sharing your code. I've been using code from uksa007 until ... . I can confirm , that your code works very well with Goodwe GW5000S-BP & GW3600S-BP.

**Comment by Sleeper85** [comment]
Thanks for the info, I'm glad it works well for you, I added it in the README.

**Comment by ghost** [comment]
Btw, above mentioned Goodwee inverters works with CAN converter TJA1050 with resistor , and also with SN65HVD230 without resistor.
